dev/conductor/core/lib/src/next.dart

```diff
@@ -194,10 +194,10 @@ class NextContext extends Context {
           previousCheckoutLocation: state.framework.checkoutPath,
         );
         stdio.printStatus('Writing candidate branch...');
-        bool needsCommit = await framework.updateCandidateBranchVersion(state.releaseVersion);
+        bool needsCommit = await framework.updateCandidateBranchVersion(state.framework.candidateBranch);
```

Comment:
I'm not really sure how to test this. Any feedback would be helpful, but i'm not sure why the version was coming out as the branch in the test.

Reply:
looking now....

Reply:
Thanks! I did `expect(stdio.stdout, contains('release-candidate-branch.version containing $candidateBranch'));` to follow the format of the existing tests so if we refactor it's a bit easier.